pkg/operation/botanist/component/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -374,13 +359,13 @@ func (c *clusterAutoscaler) computeCommand() []string {
 	return command
 }
 
-func (c *clusterAutoscaler) computeShootResourcesData() (map[string][]byte, error) {
+func (c *clusterAutoscaler) computeShootResourcesData(serviceAccountName string) (map[string][]byte, error) {
 	var (
 		registry = managedresources.NewRegistry(kubernetes.ShootScheme, kubernetes.ShootCodec, kubernetes.ShootSerializer)
 
 		clusterRole = &rbacv1.ClusterRole{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: "system:cluster-autoscaler-shoot",
+				Name: "gardener.cloud:target:cluster-autoscaler",
```

Comment:
For me to understand: Is `target` now the term to be used instead of `shoot`? What's the motivation to change this naming?

Remark: I personally prefer to stick to the terms `shoot` and `seed` because they are unambiguous whereas `source/target` highly depends on the context. For instance: Here, `target ≜ shoot` but if you look at `gardener/external-dns-management`, then `source ≜ shoot` ([ref](https://github.com/gardener/external-dns-management/#automatic-creation-of-dns-entries-for-services-and-ingresses)).

Reply:
I don't have a strong opinion, but `target` is already used at quite a few places (e.g., CA also uses `TARGET_KUBECONFIG` env variable, MCM uses https://github.com/gardener/gardener-extension-provider-aws/blob/07848722a43ec0a7e9ad79648d979b5263d16d9c/charts/internal/machine-controller-manager/seed/templates/deployment.yaml#L52, this CA component uses `managedResourceTargetName`, the KSCHED uses a similar name for the `ClusterRole`, etc.). So, can we simply keep it for now? I don't think that `gardener/external-dns-management` should have too much influence here :)

Reply:
Fair enough, then let's also change it consistently (as follow-up item) for other `ClusterRoles` like:
```
gardener.cloud:vpa:shoot:actor                                           
gardener.cloud:vpa:shoot:admission-controller                             
gardener.cloud:vpa:shoot:checkpoint-actor                                 
gardener.cloud:vpa:shoot:evictioner                                      
gardener.cloud:vpa:shoot:metrics-reader                                   
gardener.cloud:vpa:shoot:target-reader
```

Reply:
Can you comment this to https://github.com/gardener/gardener/pull/5009? I can then follow up there :)